packages/react-components/react-tags/src/components/TagButton/useTagButton.tsx

```diff
@@ -90,11 +89,10 @@ export const useTagButton_unstable = (props: TagButtonProps, ref: React.Ref<HTML
       id,
     }),
 
-    content: useARIAButtonShorthand(props.content, {
+    content: resolveShorthand(props.content, {
       required: true,
```

Comment:
yeah, it's a required slot! https://github.com/microsoft/fluentui/pull/28033/files#diff-1d0256885c4d14cf0fd3a2bb1343d5155a079229e1a77ac71613c42032fc6e0bR9

Reply:
:D yep, oops